controller/pagecontroller.php

```diff
@@ -53,6 +53,13 @@ public function index() {
 			$csp->addAllowedImageDomain('https://api.tiles.mapbox.com');
 			// inline images
 			$csp->addAllowedScriptDomain('data:');
+			// nominatim geocoder
+			$csp->addAllowedScriptDomain('http://nominatim.openstreetmap.org/search?q=*');
+			$csp->addAllowedScriptDomain('http://nominatim.openstreetmap.org/reverse');
+			$csp->addAllowedConnectDomain('http://router.project-osrm.org');
+			// mapzen geocoder
+			$csp->addAllowedConnectDomain('http://search.mapzen.com/v1/search?');
```

Comment:
Is it possible to add these CSPs depending on which geocoder is chosen.
We should reduce the allowed domains to a minimum.
cc @LukasReschke